packages/components/modules/activity-log/ActivityLogComponent/index.tsx

```diff
@@ -0,0 +1,62 @@
+'use client'
+
+import { ChangeEventHandler, FC, useState, useTransition } from 'react'
+
+import { Searchbar } from '@baseapp-frontend/design-system'
+
+import { Box, Typography } from '@mui/material'
+import { useForm } from 'react-hook-form'
+
+import SearchNotFoundState from '../../messages/SearchNotFoundState'
+import { useActivityLogs } from '../graphql/queries/ActivityLogsFragment'
+import EventFilterChip from './EventFilterChip'
+import DefaultLogGroups from './LogGroups'
+import { ActivityLogComponentProps } from './types'
+
+const ActivityLogComponent: FC<ActivityLogComponentProps> = ({
+  queryRef,
+  LogGroups = DefaultLogGroups,
+  LogGroupsProps,
+}) => {
+  const [selectedFilters, setSelectedFilters] = useState<string[]>(['All'])
```

Comment:
I think we could have defined `EVENT_FILTER_OPTIONS` and `EventFilterOption` under `ActivityLogComponent/(constants/types).ts` so we could use it on this state